.travis.yml

```diff
@@ -56,7 +54,7 @@ matrix:
         on:
           all_branches: true
   exclude:
-    - php: 7.2 # Replaced with additional tests
+    - php: 5.6 # Only run e2e on php 7.2
```

Comment:
```suggestion
    - php: 5.6 # Only run e2e with functional tests on php 7.2
```

Reply:
From what I understood e2e were the old Selenium tests and Functional were the new e2e. Since the old test will be deleted I suppose it's better just to call them e2e, don't you think?

Reply:
I don't disagree with that, but the comment you updated does not make any sense. By looking at the diff I see "Run e2e tests on PHP 5.6" while the comment says the opposite.

Reply:
> By looking at the diff I see "Run e2e tests on PHP 5.6"

Where? O_o

Reply:
I mean that's what I understand :laughing: